pkg/pipeline/ingest/metrics.go

```diff
@@ -13,16 +13,16 @@ var (
 		operational.TypeHistogram,
 		"stage",
 	)
-	batchSizeSummary = operational.DefineMetric(
+	flowsProcessed = operational.DefineMetric(
 		"ingest_flows_processed", // This is intentionally named to emphasize its utility for flows counting, despite being a batch size distribution
-		"Provides number of flows processed, batches processed, and batch size stats (in number of flows)",
-		operational.TypeSummary,
+		"Provides number of flows processed",
+		operational.TypeCounter,
 		"stage",
 	)
-	batchSizeBytesSummary = operational.DefineMetric(
-		"ingest_batch_size_bytes",
-		"Ingested batch size distribution, in bytes",
-		operational.TypeSummary,
```

Comment:
@jotak This PR reverts part of your previous work. Since we don't batch anymore at ingest time.

Another option would be keep this batchSize summary and apply it at the GRPC interceptor, so we know the batch sizes from the client. For the kafka use case, it would happen that the summary reports that every batch is size 1.

Reply:
(other option, just keep different metrics: `flowsProcessed`, `ingestBytes` for kafka ingest and `batchSizeSummary`, `batchSizeBytesSummary` for GRPC ingest.

Reply:
The idea behind these metrics was to get stats like knowing the effective size of agent's batches. As you said, it doesn't work with kafka because `kafkago` hides this detail, but that's still interesting for GRPC.
Since it's measuring ingesters input and not their output, I think removing the batches on output shouldn't affect them..

About using different metric names with or without kafka, the drawback is that it's harder to consume: we need to create more prom queries / grafana dashboards. So we need anyway to do a compromise somewhere. Personally I'd like to stick with the previous metrics.

Reply:
Sounds reasonable. Bringing them back.